chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SurveillanceController.java

```diff
@@ -157,8 +160,7 @@ public void streamDocumentContents(@PathVariable("documentId") Long documentId,
 			produces="application/json; charset=utf-8")
 	public synchronized @ResponseBody Surveillance createSurveillance(
 			@RequestBody(required = true) Surveillance survToInsert) 
-		throws InvalidArgumentsException, ValidationException, EntityCreationException, 
-		EntityRetrievalException, CertificationBodyAccessException, JsonProcessingException {
+		throws Exception {
```

Comment:
This isn't wrong, but it's preferable to have a specific list of exceptions that can be thrown by the method over the generic Exception in case the code or user that handles the errors wants to handle different types of exceptions differently. I don't think the throws clause needed to be changed in this case.